packages/worker/.env.example

```diff
@@ -23,6 +23,9 @@ RPC_CALLS_DEFAULT_RETRY_TIMEOUT=30000
 RPC_CALLS_QUICK_RETRY_TIMEOUT=500
 RPC_CALLS_CONNECTION_TIMEOUT=20000
 RPC_CALLS_CONNECTION_QUICK_TIMEOUT=10000
+RPC_BATCH_MAX_COUNT=1
```

Comment:
Most people will make this the default, is `RPC_BATCH_MAX_COUNT=1` a good default to have?

Reply:
Agreed, set batching size to 10 by default which should be a good default value.